test/manual-test-examples/webgl/pixels/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="">
+  <script language="javascript" type="text/javascript" src="../../../../lib/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="sketch.js"></script>
+  <style>
+    html, body {margin:0; padding:0;}
+  </style> 
+</head>
+<body>
+<h2 style='margin-left:100px'>If you click the canvas, the box should get textured with an image of the canvas</h2>
```

Comment:
Oh strange! When I saw this I was a bit confused as the `pointLight()` call alone should keep the box from looking like this and in Chrome the example looks like this.

![screen shot 2017-07-20 at 9 31 34 am](https://user-images.githubusercontent.com/10382506/28428625-cf967e46-6d2e-11e7-97b2-ed1aedf51f56.png)
![screen shot 2017-07-20 at 9 35 29 am](https://user-images.githubusercontent.com/10382506/28428640-d4eeb21e-6d2e-11e7-88dd-17ea395dbb3d.png)

When I saw yours I realized I forgot to check firefox. Not sure if this has something to do with pointLight. I'll investigate more.